core/src/main/java/it/pagopa/selfcare/external_api/core/UserServiceImpl.java

```diff
@@ -54,4 +59,36 @@ public UserInfoWrapper getUserInfo(String fiscalCode) {
         return result;
     }
 
+    @Override
+    public UserDetailsWrapper getUserOnboardedProductDetails(String userId, String institutionId, String productId) {
+        log.trace("getUserOnboardedProductDetails start");
+        log.debug("getUserOnboardedProductDetails userId = {}, institutionId = {}, productId = {}", userId, institutionId, productId);
+        UserDetailsWrapper result = null;
+        OnboardingInfoResponse onboardingInfoResponse = msCoreConnector.getInstitutionProductsInfo(userId);
```

Comment:
does exist any api on ms-core that return only onboarded-product for that institution? you might avoid to get entire list and filtering here...

Reply:
There arent any ready ones